woocommerce/woocommerce-gutenberg-products-block
# Migrate translation tests to Playwright

## What

Moves the translation tests for the Cart and Checkout block from Puppeteer to Playwright.

Fixes #9058

## Why

Migrating the e2e tests as we'll proceed with Playwright and will remove Puppeteer in the future.

## Testing Instructions

_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Ensure that all Playwright e2e tests are passing.


## Screenshots or screencast

n/a

## WooCommerce Visibility


## Checklist

Required:


Conditional:


## Changelog

n/a
